src/core_plugins/kibana/public/home/components/feature_directory.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Synopsis } from './synopsis';
+import {
+  KuiTabs,
+  KuiTab,
+  KuiFlexItem,
+  KuiFlexGrid,
+} from 'ui_framework/components';
+
+const ALL = 'all';
+
+export class FeatureDirectory extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    const tabs = [{
+      id: ALL,
+      name: 'All',
+    }];
+    if (props.directories.some(directory => directory.category === props.directoryCategories.DATA)) {
+      tabs.push({
+        id: props.directoryCategories.DATA,
+        name: 'Explore & Visualize'
+      });
+    }
+    if (props.directories.some(directory => directory.category === props.directoryCategories.ADMIN)) {
+      tabs.push({
+        id: props.directoryCategories.ADMIN,
+        name: 'Administrative',
+      });
+    }
+    if (props.directories.some(directory => directory.category === props.directoryCategories.OTHER)) {
```

Comment:
I think we should consider apps for the 'OTHER' tab as anything not in ADMIN or DATA, so if a plugin registers a directory in category 'stuff', it'll show up under the 'OTHER' tab.  Right now it would only show up under the 'All' tab.